src/app.py

```diff
@@ -0,0 +1,80 @@
+import gradio as gr
```

Comment:
You can add `#!/usr/bin/env python3` as shebang and check if the file is execultable `chmod +x src/app.py`.